Re-enable (and fix) terminal tests

serenity4 opened the pull request:
I fixed and refactored tests for `test_terminal.jl`, which now all pass except one where it seems that the Revise integration does not work for some reason. I marked it as broken so we can later address it.

I'm not sure about enabling/fixing IRShow tests right now, because it seems like they would very easily break by testing against hardcoded IR. I would favor having more precise tests that cover styling (colors etc) as part of the terminal tests, which we can add in the future.

serenity4 committed:
Re-enable (and fix) terminal tests

serenity4 committed:
Add test for parameter dumping ('P')

serenity4 committed:
Merge branch 'master' of github.com:JuliaDebug/Cthulhu.jl into restore-disabled-tests

aviatesk reviewed (approved):
Thanks so much for working on this!

serenity4 committed:
Define/refactor fake terminal utilities in Cthulhu

serenity4 committed:
Disable toggles for precompilation

Otherwise, we may hit a hang (a bug occurring
after a few toggles are entered), to be diagnosed
further

serenity4 committed:
Minor fix

serenity4 committed:
Fix typo

serenity4 committed:
Try not to hang in CI

serenity4 committed:
Attempt to fix tests

serenity4 commented:
@aviatesk feel free to review changes introduced in this additional commit: e262ead546465c57af23549c20f1d88d9b91bb01

If your approval still stands, I think we can merge this :)

aviatesk reviewed (approved):
Changes looks great. Thanks for working on this!
